Backpack/Toast/Classes/BPKToast.m

```diff
@@ -31,22 +31,31 @@ @interface BPKToast () <MBProgressHUDDelegate>
 @implementation BPKToast
 
 #pragma mark - Initializers
-+ (instancetype)showToastAddedTo:(UIView *)view animated:(BOOL)animated {
-    BPKToast *toast = [[BPKToast alloc] initWithView:view animated:animated];
++ (instancetype)showToastAddedTo:(UIView *)view
+                        animated:(BOOL)animated
+       accessibilityAnnouncement:(nonnull NSString *)accessibilityAnnouncement {
+    BPKToast *toast = [[BPKToast alloc] initWithView:view
+                                            animated:animated
+                           accessibilityAnnouncement:accessibilityAnnouncement];
+    [toast postAccessibilityNotification];
     return toast;
 }
 
-- (instancetype)initWithView:(UIView *)view animated:(BOOL)animated {
+- (instancetype)initWithView:(UIView *)view
```

Comment:
On second thoughts, should this method be called `showWithView` because it creates and shows the hud?

Reply:
Eventually we have to call an `init` though and that `init` has to call `showHUDAddedTo:animated:` anyway